lib/src/metta/runner/stdlib.rs

```diff
@@ -1466,6 +1466,7 @@ mod non_minimal_only_stdlib {
         (= (if False $then $else) $else)
         (: Error (-> Atom Atom ErrorType))
 
+        (= (merge-space $dst $space)  (let $x $space (add-atom $dst $x)))
```

Comment:
Should work as well because argument is reduced before call (same for minimal MeTTa):
```suggestion
        (= (merge-space $dst $space)  (add-atom $dst $space))
```